### ISSUE TITLE
Question: modify callback dump

### ISSUE THREAD

**Issue by danvargg** [scikit-optimize/scikit-optimize#810]
Hi,

This isn't an actual issue, just a question.

I have a `callback.pkl` file with ~60 iterations of a machine learning model bayesian optimization.
I need to modify the historical runs (add dimensions manually) for reasons specific to my project, and continue the iterations instead of starting them from scratch.

Is there a specific way I should load the `.pkl` to edit the content? Are there other arguments I would need to modify besides `x` and `dimensions` to include the new ones?
I've been trying to unpickle and dump as `txt` and `json` with no success.

Basically I'm trying to: `Unpickle > dump as txt / json > modify > pickle > run gp_minimize from callback`

Is this a proper course of actions? I guess my question is: can I modify the pickeled callback to add / modify / remove items in the space? OR, can I just supply `x0` and `y0` with a list manually to continue the optimization?

Great package by the way!

**Comment by danvargg** [comment]
So, I answered my own question.

I was able to pass an `ndarray` with the previous results to `y0` and a nested (modified) list to `x0`, I did not have to modify the `pkl`, just extract the data I needed and feed it to `gp_minimize`.

Keep it up!
